code/game/objects/items/mountable_frames/frames.dm

```diff
@@ -25,3 +25,16 @@
 			user << "<span class='rose'>[src] cannot be placed in this area.</span>"
 			return
 		return 1
+
+/obj/item/mounted/frame/newscaster
```

Comment:
Why would you need more than one type of frame tab open at once? And scrolling through files like vending.dm, the old pipes.dm, human/life.dm and plenty of others is just awful in every way.